### ISSUE TITLE
Not picking up ME sales

### ISSUE THREAD

**Issue by DataKnox** [milktoastlab/SolanaNFTBot#58]
Txn IDs for testing:
3poy494S99cQumQ7b7vitzgeh1PirVhCgxHgn1KkivRWJMUqXoWyLrnXwBeSyQoMUzsTRFJLrRQwHKWymv5oKZ1Y
4DP7nozRC7GBF5av2nVsz8AZ1brr9RErznuxyEHEfpDVjtAJs9hXnsHYsz6NoiWd6zcy7HoFbFvMWm7M5y4YsMt7
2RBY52vtWE5v8enniqCRPw8CaMAYik2A2mBUizrvEPt9xMkYwaSU7rnnMeREo1PchjVEtEZH2zmDCnGR4hH2gQ5x

Using the test endpoint, these all return no NFT sale found or no transaction found.

**Comment by kryptoj** [comment]
Hmm, this seems to be related to https://github.com/milktoastlab/SolanaNFTBot/issues/55 where Candy machine v2 changed the way it processes royalties which renders the parsing transaction logic useless. Have you tried the latest version? You will have to update the environment variable. https://github.com/milktoastlab/SolanaNFTBot#magic-eden-variables

**Comment by DataKnox** [comment]
Yes, the version I was building from was 5 days old from when I opened this issue. You're right that this is related to how ME handles royalties. I submitted #59 - a PR to address

The code breaks when it sees only 1 transfer of SOL, expecting there to be multiple transfers (to the seller, and to the creator for royalties)

**Comment by kryptoj** [comment]
Hrmm that's a bug. The latest version shouldn't rely on parsing the transaction anymore. New sales should get picked up by hitting MagicEden's API instead. Is that not working for you? Have you turned it on by setting the new MagicEden Env vars?

**Comment by DataKnox** [comment]
My ME variables are set and I'll add that it is not picking up sales from the Hot Heads collection. I have to remove the block that was in the PR, then hit the test endpoint with the transaction ID in order to get it to work. Unless I've configured it wrong somehow?

**Comment by kryptoj** [comment]
Interesting. What was your environment variable? 
I can see the sales activity by hitting the API using `hot_heads` as the value. 

https://api-mainnet.magiceden.dev/v2/collections/hot_heads/activities?offset=0&limit=100

**Comment by DataKnox** [comment]
ME discord channel env var is my target channel ID

ME api base url I've left unchanged

ME collection key is hot_heads

**Comment by lfmistrinel** [comment]
XD
